### PR TITLE
[UX] SSH: double info dialog timeout

### PR DESCRIPTION
**Pull request by Frenzie**
I tried to actually use this feature and five seconds isn't enough time to register the relevant IP:port to connect.

I'd add a menu item to call up the info (without a timeout) but this is just a web interface quickie.

### CODE REVIEW

**Comment by NiLuJe** [comment]
Don't we already have a Wireless info menu that print our current IP?

Because that's basically that + a user-set port with a default visible in the meny itself (IIRC) ;).

**Comment by Frenzie** [comment]
Yes, I went to the wifi info, which I found rather irksome to have to do, and then I tripped up on 2222 as I assumed 22. Besides which, I did see something 22-ish from the corner of my eye before that dialog ran away like a doe.

**Comment by Frenzie** [comment]
tl;dr I didn't assume a non-standard port because I didn't set a port.

**Comment by NiLuJe** [comment]
Fair enough ;).

I'm clearly not the right guy to judge the user-friendliness of this...

`/etc/hosts`
```
192.168.2.2     kindle
192.168.15.244  diags
192.168.0.50    kobo
192.168.0.49    forma
```

;p.

**Comment by Frenzie** [comment]
Given that it works out of the box I think it's about as user-friendly as can be, except for this "so what exactly should I enter in my FTP client" conundrum. :-P
